Aztec/Classes/TextKit/RenderableAttachmentDelegate.swift

```diff
@@ -26,3 +26,8 @@ protocol RenderableAttachmentDelegate: class {
     ///
     func attachment(_ attachment: NSTextAttachment, imageForSize size: CGSize) -> UIImage?
 }
+
+public protocol RenderableAttachment: class {
```

Comment:
Small change but, can we rename this file to `RenderableAttachment.swift`?

Reason is that now `RenderableAttachmentDelegate` is only a "detail" of `RenderableAttachment`.